shared/src/main/scala/com/raquo/domtypes/generic/defs/attrs/Attrs.scala

```diff
@@ -1,18 +1,18 @@
 package com.raquo.domtypes.generic.defs.attrs
 
-import com.raquo.domtypes.generic.builders.AttrBuilder
+import com.raquo.domtypes.generic.builders.HtmlAttrBuilder
 import com.raquo.domtypes.generic.codecs.{BooleanAsOnOffStringCodec, BooleanAsTrueFalseStringCodec}
 
-/** @tparam A HTML Attribute, canonically [[com.raquo.domtypes.generic.keys.Attr]] */
-trait Attrs[A[_]] { this: AttrBuilder[A] =>
+/** @tparam A HTML Attribute, canonically [[com.raquo.domtypes.generic.keys.HtmlAttr]] */
+trait Attrs[A[_]] { this: HtmlAttrBuilder[A] =>
```

Comment:
@raquo Maybe would be more consistent to rename Attrs to `HtmlAttrs` here? That would be same as with SvgAttrs and AriaAttrs.